gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjureJavaLocalCodegenPlugin.java

```diff
@@ -134,7 +134,7 @@ private static void createGenerateTask(
         ConjurePlugin.ignoreFromCheckUnusedDependencies(project);
 
         TaskProvider<WriteGitignoreTask> generateGitIgnore = ConjurePlugin.createWriteGitignoreTask(
-                project, "gitignoreConjure", project.getProjectDir(), ConjurePlugin.JAVA_GITIGNORE_CONTENTS);
+                project, "gitignoreConjure", project.getProjectDir(), "/build/generated/\n");
```

Comment:
I think this is still undesirable since it's generated in the wrong place?

Reply:
if this goes back to `src/generated/\n`, I'll fix up the this task so we can actually get rid of it

Reply:
Ah thanks, good catch

Reply:
I'm happy to fix this, it should just go in `/build/generated` right?

Reply:
alright thanks! yeah, should be the same as the other java tasks